tests/braille-specs/zh-tw.yaml

```diff
@@ -47,7 +47,11 @@ tests:
     - "⠢⠸⠹⠆⠌⠒⠸⠼"
     - outputPos: [0, 1]
       inputPos: [0, 1, 1, 1, 1, 1, 1, 1]
-# Han character testcases (inform the maintainer if any change to this line)
+# Han character testcases: The following testcases are generated from
+# some dictionary file(s) automatically. To facilitate maintenance with
+# the previous fixed testcases, a boundary is necessary here. Please
+# inform the maintainer if there is any change to it.
```

Comment:
What boundary?

"Please inform the maintainer if there is any change to it.": What is "it"?

Reply:
He's probably talking about some sentinel marker to indicate where the hand crafted tests end and where the tests from the dictionary file start.

Reply:
See the strings _# Han character testcases_ and _#end Han character testcases_. He is probably saying that he wants to be informed if anyone changes this

Reply:
Ah, Bo-Jeng even mentions it in the description of the PR

> Note: The following line
> `# Han character testcases (inform the maintainer if any change to this line)`
> is meant to be recognized by some script to manage thousands of testcases.

Reply:
@school510587 I would change it into the following:

```yaml
# Han character testcases: The following test cases are generated from
# some dictionary file(s) automatically. Please don't make any manual
# changes to this section (including the #begin and #end sentinel
# markers).
#begin Han character testcases
...
#end Han character testcases
```

I think this is slightly clearer.

A possibly better solution would be to put the automatically generated tests in a separate file. Then we'd still have the warning comment at the top, but we wouldn't need the sentinel markers.

Reply:
> Ah, Bo-Jeng even mentions it in the description of the PR

@egli I saw that, I'm just trying to make the comment in the test file understandable.

Reply:
Done.